client/js/widgets/widget.js

```diff
@@ -922,15 +922,28 @@ export class Widget extends StateManaged {
 
       if(a.func == 'COUNT') {
         setDefaults(a, { collection: 'DEFAULT', variable: 'COUNT' });
+        setDefaults(a, { collection: 'DEFAULT', variable: 'COUNT', owner: null });
```

Comment:
This line should be kept and the previous one removed.

Reply:
Good catch.  Fixed.